advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,586 @@
+P00   INFO: repo1: expire full backup 20210419-143400F
+P00   INFO: repo1: remove expired backup 20210419-143400F
+P00   INFO: expire command end: completed successfully
+
+$ psql -d test -c "INSERT INTO t1 VALUES (2);"
+INSERT 0 1
+$ pgbackrest backup --stanza=demo --type=full --repo=2
+P00   INFO: backup command begin ...
+P00   INFO: execute non-exclusive pg_start_backup(): backup begins after the requested immediate checkpoint completes
+P00   INFO: backup start archive = 000000010000000000000018, lsn = 0/18000028
+P00   INFO: full backup size = 63MB
+P00   INFO: execute non-exclusive pg_stop_backup() and wait for all WAL segments to archive
+P00   INFO: backup stop archive = 000000010000000000000018, lsn = 0/18000138
+P00   INFO: check archive for segment(s) 000000010000000000000018:000000010000000000000018
+P00   INFO: new backup label = 20210419-143749F
+P00   INFO: backup command end: completed successfully
+P00   INFO: expire command begin ...
+P00   INFO: repo2: expire full backup set 20210419-142414F, 20210419-142414F_20210419-142556I
+P00   INFO: repo2: remove expired backup 20210419-142414F_20210419-142556I
+P00   INFO: repo2: remove expired backup 20210419-142414F
+P00   INFO: expire command end: completed successfully
+
+$ psql -d postgres -c "select pg_create_restore_point('RP1');"
+ pg_create_restore_point 
+-------------------------
+ 0/190000C8
+(1 row)
+
+$ psql -d postgres -Atc "select current_timestamp,current_setting('datestyle'),txid_current();"
+19-APR-21 14:38:55.313716 +00:00|Redwood, SHOW_TIME|1183
+
+$ psql -d test -c "INSERT INTO t1 VALUES (3); SELECT pg_switch_wal();"
+ pg_switch_wal 
+---------------
+ 0/19000238
+(1 row)
+
+$ pgbackrest info --stanza=demo
+stanza: demo
+    status: ok
+    cipher: none
+
+    db (current)
+        wal archive min/max (13): 000000010000000000000016/000000010000000000000019
+
+        full backup: 20210419-143643F
+            timestamp start/stop: 2021-04-19 14:36:43 / 2021-04-19 14:37:01
+            wal start/stop: 000000010000000000000016 / 000000010000000000000016
+            database size: 63.0MB, database backup size: 63.0MB
+            repo1: backup set size: 10MB, backup size: 10MB
+
+        full backup: 20210419-143749F
+            timestamp start/stop: 2021-04-19 14:37:49 / 2021-04-19 14:38:03
+            wal start/stop: 000000010000000000000018 / 000000010000000000000018
+            database size: 63MB, database backup size: 63MB
+            repo2: backup set size: 10MB, backup size: 10MB
+```
+
+As you can see in the example above, the `backup` command automatically triggers the `expire` command on the repository where we took our backup. That means it won't expire anything in the other repositories unless you request a new backup there or execute the `expire` command manually.
+
+Since the default retention policy is based on a number of backups, it is safe and healthy that only new backups are affecting the backups and archives to expire.
```

Comment:
I don't understand what you are trying to say here.
Do you mean 'expected' instead of 'safe and healthy'?

Reply:
Yes, that's the expected behavior but that would also be a good (safe and healthy) recommendation... 
Perhaps change to "it is safe and expected" rather than healthy ?

Reply:
Since the default retention policy is based on a number of backups, it is safe and healthy that only new backups are affecting the backups and archives to expire.

Reply:
Safe and healthy sounds defensive. Replace with 'expected'.

Reply:
Fixed in 7fa8ff1